AlfrescoContent/AlfrescoContent/Classes/Swaggers/Process/ProcessRequestLinkContent.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ProcessRequestLinkContent.swift
+//  AlfrescoContent
+//
+//  Created by Ankit Goyal on 31/03/23.
+//
+
+import Foundation
+
+public struct ProcessRequestLinkContent: Codable {
+    public var source: String?
+    public var mimeType: String?
+    public var sourceId: String?
+    public var name: String?
+
+    public init(source: String? = "alfresco-1-adw-contentAlfresco",
```

Comment:
As we've fixed source value for now but it must be passed from the client app only. Please remove the default value of source.

Reply:
Its handled in a way that it can be passed from the client app as well. It will override the default value if its passed from there else this default value be used.